invenio_oaiserver/config.py

```diff
@@ -133,3 +133,6 @@
 
 OAISERVER_CELERY_TASK_CHUNK_SIZE = 100
 """Specify the maximum number of records each task will update."""
+
+OAISERVER_CONTROL_NUMBER_KEY = 'control_number'
```

Comment:
Are we sure this number would always be part of some other PID as well? Additionally, this would impose order in which PIDs are minted.